inc/notificationtargetorder.class.php

```diff
@@ -464,35 +462,35 @@ public static function uninstall()
                 'event'    => $event,
                 'FIELDS'   => 'id',
             ];
-            foreach ($DB->request('glpi_notifications', $options) as $data) {
+            foreach ($DB->request(['FROM' => 'glpi_notifications'], $options) as $data) {
```

Comment:
`request()` doesn't have just one parameter?! I'm surprised that phpstan doesn't raise an error here.

<img width="985" height="331" alt="image" src="https://github.com/user-attachments/assets/7eb9ba6a-7a5d-48f5-b2a4-ad4f8b71c532" />

Reply:
I only pointed out one case, but there are others.